repository-structure.md

```diff
@@ -0,0 +1,72 @@
+# Common Repository Structure
+
+Hyperledger projects are required to maintain a standard set of files in each repository. This
+document describes the required and recommended files.
+
+## Required Files with Specified Content
+
+Repositories MUST have these files with the specific content in the linked files, or a file with a
+link to the specified content with minimal exposition. These files MUST be at the root of the
+repository.
+
+-   [`LICENSE`](https://www.apache.org/licenses/LICENSE-2.0.txt) - https://www.apache.org/licenses/LICENSE-2.0.txt
```

Comment:
There can be a repository which is not `Apache 2.0`. For example, an exception as per the TSC charter or `Creative Commons` for docs. Is it better to redirect readers to the appropriate locations elsewhere to read elaborately on allowed licenses? However, a `LICENSE` file can be mandated here.

Reply:
This requires an act of the governing board.  Added in that the exception may occur.